plugins/mac-virtualcam/src/dal-plugin/Defines.h

```diff
@@ -0,0 +1,21 @@
+//
+//  Defines.h
+//  obs-mac-virtualcam
+//
+//  Created by John Boiles  on 5/27/20.
+//
+//  obs-mac-virtualcam is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License as published by
+//  the Free Software Foundation, either version 2 of the License, or
+//  (at your option) any later version.
+//
+//  obs-mac-virtualcam is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with obs-mac-virtualcam. If not, see <http://www.gnu.org/licenses/>.
+
+#define PLUGIN_NAME @"mac-virtualcam"
```

Comment:
Any strong reason you moved to a static `Defines.h` file instead of sticking with the generated one? Not a huge deal either way, but kinda nice to not have to update the version number in as many places.

But as I write that, updating a few extra lines is probably worth reducing complexity of the CMake build. So I guess I don't have a strong feeling.

Reply:
Only reason was the idea that its version would be OBS' version (as it's bundled and all) and I just wanted it to compile.. Was actually thinking about removing the Defines import altogether for that reason. 😄

Reply:
Oh yeah it's a better idea to just use OBS's version for this string. I assume that is already defined in the build somewhere -- maybe I can use that instead of this code and to generate the .plist file.